Sources/MediaViewer/Transition/Presenter.swift

```diff
@@ -1,18 +1,30 @@
 import UIKit
 
+@MainActor
 final class Presenter: NSObject, UIViewControllerTransitioningDelegate {
     var interactiveTransition: InteractiveTransition? = nil
     var reversedDismiss: Bool = false
+    var onDismissed: (() -> Void)?
```

Comment:
Can any of these be private/private(set), or even "let" since you are setting them in the initializer? Can other vars that you've added in this PR be private/private(set)?